doc/nrf/app_dev/device_guides/nrf52/index.rst

```diff
@@ -20,43 +20,43 @@ Zephyr and the |NCS| provide support and contain board definitions for developin
      - PCA number
      - Board target
      - Documentation
-   * - :ref:`zephyr:nrf52840dk_nrf52840`
+   * - :zephyr:board:`nrf52840dk`
      - PCA10056
      - ``nrf52840dk/nrf52840``
      - | `Product Specification <nRF52840 Product Specification_>`_
        | `User Guide <nRF52840 DK User Guide_>`_
-   * - :ref:`zephyr:nrf52840dk_nrf52811`
+   * - :zephyr:board:`nrf52840dk`
```

Comment:
@gmarull @jfischer-no could :zephyr:board: link to nrf52840dk_nrf52811 (as was before - :ref:`zephyr:nrf52840dk_nrf52811`) ? Otherwise as seen in the preview we will have multiple entries for nRF52840 DK and other DKs (https://ncsdoc.z6.web.core.windows.net/PR-22073/nrf/app_dev/device_guides/nrf52/index.html). Here is the current state where the entries are unique depending on the section header where it is linking to: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf52/index.html

Reply:
This is the difference in the preview: 
![image](https://github.com/user-attachments/assets/1ffb1e10-1fa9-46d2-9846-88c4686bc036)
compared to the current state in the docs:
![image](https://github.com/user-attachments/assets/23c6ae4a-a091-4fec-83bd-e6bdd6502916)

Reply:
If not, we could keep the link as is as - :ref:`zephyr:nrf52840dk_nrf52811`, so that it links to the [nRF52811 emulation on nRF52840 DK](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/nordic/nrf52840dk/doc/index.html#nrf52840dk-nrf52811) section, as expected.

Same applies for zephyr:nrf52dk_nrf52810 (line no:46) and zephyr:nrf52dk_nrf52805 (line no:50).

Reply:
No, boards are now referenced by their name, not target. It actually makes more sense. Maybe adjust tables to merge some cells.

Reply:
Ok. Then we will add a fix after the upmerge.

Reply:
I actually see that nrf9160dk_nrf52840 tags exist, so ref can be used. Feel free to amend these cases from my patch if you think it is better.